app/code/Magento/SalesRule/view/frontend/web/js/view/summary/discount.js

```diff
@@ -44,6 +44,23 @@ define([
             return this.totals()['coupon_label'];
         },
 
+        /**
+         * Get discount title
+         *
+         * @returns {null|string}
+         */
+        getTitle: function () {
+            if (!this.totals()) {
+                return null;
+            }
+
+            var discountSegments = this.totals()['total_segments'].filter(function (segment) {
+                return (segment.code === 'discount');
```

Comment:
I guess, the parenthesis is redundant here. This fact is the reason of failing static tests